### PR TITLE
Enabled validation.php contributions for attributes

### PR DESCRIPTION
**Pull request by kdmnk**
Also fixed bug (missing escape characters)

### CODE REVIEW

File: app/Console/Commands/AddExpression.php [hunk 1]
```diff
@@ -99,17 +104,31 @@ public function handle()
...
-                        return;
+                            return;
+                        }
+                    }
+                } else {
+                    if (isset($expressions[$expression_key])) {
+                        if (! $this->confirm('Do you want to override the old '.($language == 'en' ? 'english' : 'hungarian').' translation ('.$expressions[$expression_key].')?')) {
+                            $this->info('Action cancelled.');
+
+                            return;
```
Comment (luksan47) [inline app/Console/Commands/AddExpression.php:120]: For later: we might not want to return here, if we only wish to update the translation for the other language.

File: resources/lang/de/validation.php [hunk 2]
```diff
@@ -9,37 +9,7 @@
     'alpha_dash' => ':attribute darf nur aus Buchstaben, Zahlen, Binde- und Unterstrichen bestehen.',
     'alpha_num' => ':attribute darf nur aus Buchstaben und Zahlen bestehen.',
     'array' => ':attribute muss ein Array sein.',
-    'attributes' => [
```
Comment (luksan47) [inline resources/lang/de/validation.php:12]: I meant to comment here earlier:
Why are you removing these? How will we migrate these into the database?

Comment (kdmnk) [reply resources/lang/de/validation.php:12]: These are downloaded from the web, so these are not synced with our en/hu translations

Comment (luksan47) [reply resources/lang/de/validation.php:12]: Okay

Comment (luksan47) [reply resources/lang/de/validation.php:12]: But isn't it true for the Hungarian translations?

**Review by luksan47 — APPROVED** [review]

**Pull request merged**